installation_and_upgrade/requirements.txt

```diff
@@ -0,0 +1,9 @@
+contextlib2==21.6.0
+GitPython==3.1.41
+kafka_python==2.0.2
+lxml==4.9.3
+psutil==5.9.5
+pytest==8.0.0
+pywin32==306
+semantic_version==2.10.0
+six==1.16.0
```

Comment:
Since your adding specific requirements other than the default system ones this should probably use a venv.

Reply:
This txt contains the requirements of the ibex_utils repository (apart from pytest which is only or testing)